profiles/divina.md

```diff
@@ -210,11 +211,7 @@ As an alternative, the manifest can also be added to an EPUB ([as defined in the
 
 * TBD
 
-## Appendix B. Implementation Guidelines
```

Comment:
This PR silently deletes a link to the Divina implementation guidelines. While I agree that the status of such guidelines is not clear and must be worked on, it should not be part of this PR.

Reply:
I'll bring this back, but the usefulness of these guidelines at this stage is dubious at best.